### PR TITLE
Updated Typos in README.md

### PR DESCRIPTION
**Pull request by kamleshjoshi8102**
#### Other comments
Fixed some typos in README.md

#### Release Notes

NO ENTRY

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -270,7 +270,7 @@ A BibTeX entry for LaTeX users is
 ```
 
 SymPy is BSD licensed, so you are free to use it whatever you like, be
-it academic, commercial, creating forks or derivatives, as long as you
+it academic, commercial, creating forks, or derivatives, as long as you
```
Comment (smichr) [inline README.md:273]: I think you can leave this one out since "creating" is the action for both of them.

**Comment by smichr** [comment]
Maybe you could include #21664, too.

**Comment by kamleshjoshi8102** [comment]
> Maybe you could include #21664, too.



> Maybe you could include #21664, too.

okay sir added : )

**Comment by namannimmo10** [comment]
> okay sir added : )

You should make another commit which will fix the typo mentioned in #21664

**Comment by kamleshjoshi8102** [comment]
> > okay sir added : )
> 
> You should make another commit which will fix the typo mentioned in #21664

Sir I can't understand what you said can you explain a little bit because I added the issue number in the above PR

**Comment by mostlyaman** [comment]
Please add this typo from the docstring of collect function at sympy/simplify/radsimp.py in this PR too.
https://github.com/sympy/sympy/blob/2bbb4a1faca9b09247cd36be76b0676a4f5d3f9d/sympy/simplify/radsimp.py#L34-L35

Change  "provide an expression is an appropriate form" to "provide an expression in an appropriate form" in line 35.

**Comment by smichr** [comment]
I was hoping to get the other typo in on the same build. Nevermind, now, since a second run will have to be made anyway. So this is in. Thanks.

**Comment by kamleshjoshi8102** [comment]
> I was hoping to get the other typo in on the same build. Nevermind, now, since a second run will have to be made anyway. So this is in. Thanks.

Thank you sir ! : )

**Pull request merged**